### DISCUSSION TITLE
development

### DISCUSSION THREAD

**Discussion by AMENOUSSI** [community/community#57475] [GitHub Education]
### Hi everyone,

Hi every body, I am Jules,and I am still be student in software engineering and i like php / laravel for the web development and flutter for mobile develop
ment?. I would like to have the advice of some developers here, thanks

**Comment by AbdiRabbani** [comment]
Hi @AMENOUSSI My name is Abdi im a junior software engineering student from Indonesia, my suggestion maybe you can focused to one goal first like maybe you can be web development first an after you advance or maybe expert in website it will easier to learn mobile after that

**Reply by AMENOUSSI** [reply]
Hi @AbdiRabbani  i very glad for your contribution, it's like God sent you to advice me,for that i thank you more. Write now i am focusing on we delopment wiith larvel and vueJs, and after i will add Flutter

**Reply by AMENOUSSI** [reply]
So, since i want to attend a higher school of computer science in software  field for my Master degree, i want you to tell me about Indonesia school of computer science specialy for software engineering. What  can be at all , the school fees, restauration, and so. I am in Togo, i am not good in english but i will ask you to apologize me for my speaking.I am looking forward to hearing from very soon
